src/Services/CategoriesService.js

```diff
@@ -1,8 +1,25 @@
 import axios from 'axios';
+import es from 'date-fns/locale/es';
 
 const CATEGORIES_URL = 'http://ongapi.alkemy.org/api/categories';
 
-const createCategorie = async (categorie) => {
+const getCategoryOrCategories = async (categorieId) => {
```

Comment:
the title should be getCategories. It return 1 or more than 1 but its getCategories